src/commands/remember/basicRemember.js

```diff
@@ -35,12 +36,11 @@ module.exports = {
             //specific channel is true, search that channel
             if(specificChannelId){
                 //get the message from given channel
-                const channel = await client.channels.cache.get(specificChannelId);
-                msg =  await channel.messages.fetch(idMessage);
+                msg =  await getMessageObject(specificChannelId,idMessage);
             }
             else{
                 //get the message from current channel
-                msg =  await interaction.channel.messages.fetch(idMessage);
+                msg =  await getMessageObject(interaction.channel.id, idMessage);
```

Comment:
This can be shortened with the use of a ternary operator
```js
const specificChannelId = interaction.options.getString('search-specific-channel') ?? interaction.channel.id
msg =  await getMessageObject(specificChannelId,idMessage);
```